docs/source/fitting/fitfunctions/FlatTopPeak.rst

```diff
@@ -0,0 +1,22 @@
+.. _func-FlatTopPeak:
+
+===========
+FlatTopPeak
+===========
+
+.. index:: FlatTopPeak
+
+Description
+-----------
+
+A flat top peak function is defined as:
```

Comment:
I think it's also helpful to add a picture here as is done elsewhere e.g.
https://docs.mantidproject.org/nightly/fitting/fitfunctions/BackToBackExponential.html